src/method/method_invoke_java.cpp

```diff
@@ -635,6 +657,61 @@ namespace cubmethod
     return error;
   }
 
+  int
+  method_invoke_java::callback_make_outresult (cubthread::entry &thread_ref, cubmem::block &blk)
+  {
+    int error = NO_ERROR;
+#if defined (SERVER_MODE)
+    packing_unpacker unpacker;
+    unpacker.set_buffer (blk.ptr, blk.dim);
+
+    int code;
+    uint64_t query_id;
+    unpacker.unpack_all (code, query_id);
+
+    INT64 id = (INT64) this;
+    cubmethod::header header (METHOD_REQUEST_CALLBACK /* default */, id);
+    error = method_send_data_to_client (&thread_ref, header, code, query_id);
+    if (error != NO_ERROR)
+      {
+	return ER_FAILED;
+      }
+
+    auto get_make_outresult_info = [&] (cubmem::block & b)
+    {
+      packing_unpacker unpacker (b.ptr, (size_t) b.dim);
+
+      int res_code;
+      make_outresult_info info;
+      unpacker.unpack_all (res_code, info);
+
+      query_result_info &current_result_info = info.qresult_infos[0];
+      const auto &iter = m_cursor_map.find (current_result_info.query_id);
+      if (iter == m_cursor_map.end ())
+	{
+	  query_cursor *cursor = nullptr;
+	  // not found, new cursor is created
+	  bool is_oid_included = current_result_info.include_oid;
+	  cursor = m_cursor_map[query_id] = new (std::nothrow) query_cursor (m_group->get_thread_entry(), query_id,
+	      is_oid_included);
+	  if (cursor == nullptr)
+	    {
+	      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, sizeof (query_cursor));
+	      return ER_OUT_OF_VIRTUAL_MEMORY;
+	    }
+
+	  cursor->open ();
+	}
+
+      error = method_send_buffer_to_java (m_group->get_socket(), b);
+      return error;
+    };
```

Comment:
If there is no need to use ';', it would be better to delete it.

Reply:
ok, I'm gonna fix it